packages/flutter_analysis_options/lib/analysis_options_base.yaml

```diff
@@ -0,0 +1,77 @@
+# Specify analysis options.
+#
+# Until there are meta linter rules, each desired lint must be explicitly enabled.
+# See: https://github.com/dart-lang/linter/issues/288
+#
+# For a list of lints, see: http://dart-lang.github.io/linter/lints/
+# See the configuration guide for more
+# https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
+#
+# This file contains the analysis options used by "flutter analyze" and the
+# dartanalyzer when analyzing code outside the flutter repository. It isn't named
+# 'analysis_options.yaml' because otherwise editors would use it when analyzing
+# the flutter tool itself.
+#
+# When editing, keep in mind that this is imported by analysis_options.yaml.
+# This file is meant to be a less strict subset of rules appropriate for most
+# Flutter projects, but does not contain all of the lints that are used by the
+# core Flutter repositories.
+
+analyzer:
+  errors:
+    # treat missing required parameters as a warning (not a hint)
+    missing_required_param: warning
+
+linter:
+  rules:
+    # these rules are documented on and in the same order as
+    # the Dart Lint rules page to make maintenance easier
+    # https://github.com/dart-lang/linter/blob/master/example/all.yaml
+    # - always_declare_return_types
+    # - always_specify_types
+    # - annotate_overrides
+    # - avoid_as
+    - avoid_empty_else
+    - avoid_init_to_null
+    - avoid_return_types_on_setters
+    - await_only_futures
+    - camel_case_types
+    - cancel_subscriptions
+    # - close_sinks # not reliable enough
+    # - comment_references # we do not presume as to what people want to reference in their dartdocs
+    # - constant_identifier_names # https://github.com/dart-lang/linter/issues/204
+    - control_flow_in_finally
+    - empty_constructor_bodies
+    - empty_statements
+    - hash_and_equals
+    - implementation_imports
+    # - invariant_booleans
+    # - iterable_contains_unrelated_type
+    - library_names
+    # - library_prefixes
+    # - list_remove_unrelated_type
+    # - literal_only_boolean_expressions
+    - non_constant_identifier_names
+    # - one_member_abstracts
+    # - only_throw_errors
+    # - overridden_fields
+    - package_api_docs
+    - package_names
+    - package_prefixed_library_names
+    - prefer_is_not_empty
+    # - prefer_mixin # https://github.com/dart-lang/language/issues/32
+    # - public_member_api_docs
+    - slash_for_doc_comments
+    # - sort_constructors_first
+    # - sort_unnamed_constructors_first
+    - super_goes_last
+    - test_types_in_equals
+    - throw_in_finally
+    # - type_annotate_public_apis # subset of always_specify_types
+    - type_init_formals
+    # - unawaited_futures
+    - unnecessary_brace_in_string_interps
+    - unnecessary_getters_setters
+    - unnecessary_statements
+    - unrelated_type_equality_checks
+    - valid_regexps
```

Comment:
We should perhaps remove commented lints in this file. I don't understand if there are here as a wish to be enabled at some point but in this case we should add a comment to express that.

Reply:
Is it to make it easier to compare with the master list?

Reply:
they don't appear to be in sync if it is.

Reply:
Removed all except one that had an additional remark next to it

Reply:
The comment of `type_annotate_public_apis` reference a lint not present in the current analysis_option. I think even this one could be removed.

Reply:
Done